python/setup.py

```diff
@@ -179,6 +179,8 @@ def _supports_symlinks():
                   'pyspark.ml.linalg',
                   'pyspark.ml.param',
                   'pyspark.sql',
+                  'pyspark.sql.avro',
```

Comment:
Just a question. Is this `avro` changed required in this Pandas refactoring PR?

Reply:
Ah, no. All packages had to be added and `pyspark.sql.avro` was missing; otherwise, avro package wouldn't be available in PySpark installed by pip. I fixed it together while I am here.